### ISSUE TITLE
RFTools builder running into chunk not available issue

### ISSUE THREAD

**Issue by Sezylrin** [Divine-Journey-2/Divine-Journey-2#707]
### Modpack version

2.15

### Description

RFTools builders have a force chunkload ticket leak which results in builders to stop working in whatever current dimension it is situated in eventually. ftb Force Chunkload only allows mods to submit a chunkload ticket containing 25 chunks at a time a maximum of 200 times, anymore tickets past that in the dimension will result in builders to stop working. This results in the chunks not available message that will appear on builders for certain people
![2022-08-02_17 27 01](https://user-images.githubusercontent.com/79730008/182317091-05bd0b08-da12-41d8-9733-f0081961508b.png)

### Steps to reproduce

1. place a total of 5 builders
2. let it quarry a maximum space of 512x512 each in order to reach the ticket limit
3. Further builders in the dimension will no longer work

### Expected behavior

Builders should still work and the ticket leak should somehow be cleared or ticket limit extended to provide more time before overflow.

Changing configs for forgeChunkLoading.cfg and creating a new override for rftools can extend the ticket limit and prevent the issue happening after 5 builders with max quarries but is not a perfect solution
``` 
##########################################################################################################
# forge
#--------------------------------------------------------------------------------------------------------#
# Sample mod specific control section.
# Copy this section and rename the with the modid for the mod you wish to override.
# A value of zero in either entry effectively disables any chunkloading capabilities
# for that mod
##########################################################################################################

forge {
    # Maximum chunks per ticket for the mod.
    I:maximumChunksPerTicket=25

    # Maximum ticket count for the mod. Zero disables chunkloading capabilities.
    I:maximumTicketCount=200
}


ftbutilities {
    I:maximumChunksPerTicket=1000000
    I:maximumTicketCount=1000000
}


### Additional context

_No response_

**Comment by Preta-Crowz** [comment]
Can not reproduce, it works right on my server (confirmed on two servers and both works)
Are you on singleplayer or server?
If you are playing on server, then check your server software.

**Comment by Preta-Crowz** [comment]
Oh, I misread. But are 5 builders with max range needed?

**Comment by Lachey789** [comment]
> Oh, I misread. But are 5 builders with max range needed? If you still need to use them all, then you can change config manually.

Hey, it's an issue on a public server as the dimension is including the 5 builder max range (or a # of builders with smaller sizes adding up to the chunk limit) for all players. Currently on our server, the deep dark has reached the chunk limit meaning any one placing a new builder won't be able to use it properly as it can't load the chunks it's trying to dig.

Believe there may also be an issue where even after the builder is finished/destroyed, it's still not clearing the chunk ticket meaning further builders in the dimension aren't going to be able to chunkload

**Comment by Preta-Crowz** [comment]
> > Oh, I misread. But are 5 builders with max range needed? If you still need to use them all, then you can change config manually.
> 
> Hey, it's an issue on a public server as the dimension is including the 5 builder max range (or a # of builders with smaller sizes adding up to the chunk limit) for all players. Currently on our server, the deep dark has reached the chunk limit meaning any one placing a new builder won't be able to use it properly as it can't load the chunks it's trying to dig.
> 
> Believe there may also be an issue where even after the builder is finished/destroyed, it's still not clearing the chunk ticket meaning further builders in the dimension aren't going to be able to chunkload

But anyway, that config is for reduce server resource and etc. If server reached ticket limit and machines are stop, then admins should allow to use more tickets.
If not, that's what they choiced, it's not issue about mod or modpack.

**Comment by Lachey789** [comment]
Yes that is true, but if the builders aren't clearing tickets once the builder has completed/after being removed, this would be an issue with the mod would it not? Wouldn't the expected behaviour be for a ticket to be destroyed after the builder is destroyed to allow for builders to still be functional?

**Comment by Preta-Crowz** [comment]
Ok, confirmed [this issue](https://github.com/McJtyMods/RFTools/issues/1404) on RFTools but McJty no longer develop on 1.12

**Comment by KleinGrrmpf** [comment]
So, I have run into this issue as well and from researching it a bit by looking through github issues, all the instances seem to have happened in the nether, so it might be restricted to that dimension.
Have you previously been able to fix this @ByThePowerOfScience ? Because if you have, then your fix is not working.
